microsoft-edge/webview2/how-to/publish-uwp-app-store.md

```diff
@@ -0,0 +1,77 @@
+
+
+1. Make sure you have a Microsoft Store account configured in Microsoft Visual Studio.  See [Get registered](https://developer.microsoft.com/microsoft-store/register/), about developer accounts for the Microsoft Store.
+
+1. Make sure your application runs correctly on startup, and works correctly during shutdown sleep.
+
+1. Update the `package.appxmanifest` with all needed content, such as Visual Assets, Packaging name, and Application information.  If you don't do this, the Microsoft Store validation step will fail.
+
+1. At Partner Center, select **Project** > **Publish** and then reserve an application name.
+
+1. Associate the application with the reserved name, as follows: **Project** > **Publish** > **Associate App with the Store**.
+
+1. Create the application package: **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Only 64-bit versions of the WebView2 Runtime will work in UWP:
+
+   ![Package Selection](publish-uwp-app-store-images/package-selection.png)
+
+1. Once the application package has been created for your configurations, you'll be presented with a dialog of information and have the option to run Windows App Certification Kit (WACK):
+
+   ![Windows App Certification Kit](publish-uwp-app-store-images/win-app-cert-kit.png)
+
+1. Note where the bundle was placed (also in the build window) and **Cancel** this dialog box.  You'll run Windows App Certification Kit (WACK) after validating the package.
+
+1. Use the generated application package to install the app on your machine.  Look for any oddities regarding required packages and functionality.
+
+1. If possible, install your app on other machines also.  Look for any oddities regarding required packages and functionality.
+
+   Once you're satisfied that your packaged app works, it is time to go through Windows App Certification Kit (WACK).
+
+1. Generate the app package again, because that is the easiest way to get to the Windows App Certification Kit (WACK) process.
+
+1. Once the package creation is complete, select **Launch Windows App Certification Kit**.
+
+1. You must approve the User Access Control (UAC) dialog, for the Windows App Certification Kit (WACK) to run correctly.
+
+1. Select **All** for the test suite, and then click **Next**.  The validation will take a few minutes.
+
+1. Once complete, the Windows App Certification Kit (WACK) will show a results page.  If the application failed, click the link to review the results.
+
+   *  If your app failed the **restricted name-space** check, edit your `package.appxmanifest` file and remove the `restrictedcapabilities` line and any capabilities that use `rescap`.
+
+      WebView2 shouldn't require any restricted capabilities, to operate.  NOTE: If your application needs restricted capabilities, you'll need to either ask for a store exception through Partner Center, or only distribute the app through non-Store avenues.
```

Comment:
Should this be formatted as a note? I'm not sure about an all caps NOTE in plain text.

Reply:
Trying bold **Note:** in context of heavily reformatted page.